README.md

```diff
@@ -9,6 +9,17 @@ has changed. This is very bad and screen reader users hear nothing but silence.
 The goal for this addon is to fill the gaps in embers router. Thank you to
 @patrickfox [for the idea!](https://vimeo.com/117614181)
 
+Compatibility
+------------------------------------------------------------------------------
+
+* Ember.js v3.8 or above
+* Ember CLI v2.13 or above
+* Node.js v8 or above
```

Comment:
Just curious (and totally nit picky!) should this be bumped from v8 since support was dropped Jan 1st, 2020? https://github.com/nodejs/Release

Reply:
@rwjblue can probably tell us the reason it's still in there, but I forget exactly what the reason is.

Reply:
Ya, its because our official policy is that we support "all current Node LTS versions on the master branch at all times". In practice this means that ember-cli@3.15 (and 3.16) will continue to list Node 8 as supported.

tldr; if this PR will be considered part of a new major bump, I would drop Node 8 here before doing that